packages/core/strapi/lib/core/registries/cron.js

```diff
@@ -0,0 +1,42 @@
+'use strict';
+
+const cron = require('node-schedule');
+const { isFunction, getOr } = require('lodash/fp');
+
+const cronRegistry = () => {
```

Comment:
Just some semantic, but I don't think this should be a registry but a service in the ./services folder instead. 
The difference is that when you add a cron you actually run when in the registries you only "store" the things.
I would have created a cronRunner like the webhooksRunner instead.

Reply:
Totally agree